terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars

```diff
@@ -32,3 +32,4 @@ feature_flag_disable_update_private_data_fetch_28062023 = "false"
 feature_flag_enable_update_manage_trusts_29082023 = "true"
 feature_flag_enable_redis_removal_27092023 = "true"
 feature_flag_enable_cease_trusts_19022024 = "false"
+feature_flag_enable_relevant_period_26042024 = "false"
```

Comment:
should not this be true? as we want the changes see in development, staging env.

Reply:
potentially, I'm not a Terraform expert, so at the moment I just went for turned off in all envs except our local. Probably we can ask dev ops to turn these on as and when we need to use each for testing.

Reply:
With the terraform approach on the web services, the pipeline will pick up the terraform from your PR and push it through and apply it to CI-Dev. When the next release goes into Staging and live, it takes the latest terraform with it. So if you leave it as false, but want it set to true for staging, you would need to create a new PR to set it to true when you are ready to go to staging.
For the services that haven't migrated to these new Amazon ECS pipelines (like overseas-entities-api) you would request the release team manually change the config in staging and live via release notes.
The Pr looks ok to me if you are happy with the config values.